iree/compiler/Dialect/HAL/IR/LoweringConfig.h

```diff
@@ -144,6 +151,13 @@ inline SmallVector<int64_t, 4> getNativeVectorSize(Operation *op) {
   return getNativeVectorSize(configAttr);
 }
 
+/// Get the pass pipeline specified in the `loweringConfig`
+inline Optional<IREE::HAL::DispatchLoweringPassPipeline>
+getLoweringPassPipeline(IREE::HAL::LoweringConfig config) {
```

Comment:
Actually just noticed that we also have the `passPipeline` on `lowering.config`. Why having both? That's a duplication with `translation.info`? Checking the tests changed it looks only some LLVM CPU tests use it. Can we change LLVM CPU side to rely on `translation.info` and remove the duplication?

Reply:
It is needed in both. The one in `lowering.config` is used for when you want to control the pipeline to use from say MHLO. So when you set the `lowering.config` on a `mhlo.dot` that will be used instead of using the what is picked by the `init*` method of the backend.
In any case, that is unrelated to this PR.

Reply:
We are cleaning up this so it's worth to clean up that too. But I don't know that actually they are meant for different things. SGTM then.